packages/webpack-config/src/helpers/paths.js

```diff
@@ -62,9 +58,29 @@ export const appTarget = resolveApp(config.destPath);
 
 const getAppName = () => {
   const { namespace } = getAppConfig();
-  if (namespace ==='') return 'app';
-  
+  if (namespace === "") return "app";
+
   return `${namespace}.app`;
-}
+};
+
+export const appName = getAppName();
+
+// check if a hbs partial dir is provided
+export const hbsPartialDir = {
+  partialDirs: config.hbsPartialDir ? [path.resolve(config.hbsPartialDir)] : []
+};
+
+/******************************************************************************
+ ** CompileHTML helper
+ ******************************************************************************/
+
+const getRequiredHtmlCompilerConfig = () => {
+  return Boolean(config.hbsEntry && config.hbsTarget);
+};
+export const useHtmlCompiler = getRequiredHtmlCompilerConfig();
+export const hbsEntry = useHtmlCompiler ? path.resolve(config.hbsEntry) : "/";
+export const hbsTarget = path.resolve(`${appTarget}/${config.hbsTarget}`);
```

Comment:
I think hbsTarget should be the full target path. There could be a a usecase for rendering the hbs files to a different target directory.

Reply:
done